scripts/deploy.js

```diff
@@ -0,0 +1,13 @@
+const deployTemplate = require("@aragon/templates-shared/scripts/deploy-template");
+
+const TEMPLATE_NAME = "dandelion-org-template";
+const CONTRACT_NAME = "DandelionOrg";
+
+module.exports = callback => {
+  deployTemplate(web3, artifacts, TEMPLATE_NAME, CONTRACT_NAME)
+    .then(template => {
+      console.log(template.address);
+      callback();
+    })
+    .catch(callback);
```

Comment:
Why not use async await? Also what is the callback for/how is this called?

Reply:
After looking at how the doc looks (since it turns out I've been using it wrong :p), I would rewrite this as:
```
module.exports = async callback => {
try {
    const template = await deployTemplate(web3, artifacts, TEMPLATE_NAME, CONTRACT_NAME)
    console.log(template.address)
} catch (error) {
    callback(error)
}
callback()
```
I'd put the whole script inside the `try` to catch any errors. Up to y'all though!

Reply:
lol! yes sure i can move it to async await